libraries/RC_Channel/RC_Channel.h

```diff
@@ -281,7 +281,7 @@ class RC_Channel {
     const char *string_for_aux_function(AUX_FUNC function) const;
 #endif
     // pwm value under which we consider that Radio value is invalid
-    static const uint16_t RC_MIN_LIMIT_PWM = 900;
+    static const uint16_t RC_MIN_LIMIT_PWM = 800;
```

Comment:
This looks OK but I can't resist saying that this constant's name should really be, "RC_PWM_LIMIT_MIN".  Wouldn't that be better?  Anyway, no need to change it I just want to pass on my naming wisdom.

Reply:
`LIMIT` is kind of inferred by `MIN` :-)

Reply:
Yes, removing the LIMIT part would also be a fine change.. but not today I'm sure..